app/view/templates/pages/upload/step2/_custom_strain_form.html

```diff
@@ -28,9 +28,7 @@
               required>
             {% if submission.studyDesign['custom_strains']|length > 0 and index is not none and index < submission.studyDesign['custom_strains']|length: %}
               {% set custom_strain_data = submission.studyDesign['custom_strains'][index] %}
-              <option value="{{ custom_strain_data['species'] }}">
-              {{ custom_strain_data['species_name'] }} (NCBI:{{ custom_strain_data['species'] }})
-              </option>
+              <option value="{{ custom_strain_data['species'] }}">{{ custom_strain_data['species_name'] }} (NCBI:{{ custom_strain_data['species'] }})</option>
```

Comment:
Option tags should not have any whitespace inside the tag, or, annoyingly, it gets picked up by select2.